aea/cli/remove.py

```diff
@@ -373,36 +386,9 @@ def agent_name(self) -> str:
 
     def _get_item_folder(self) -> Path:
         """Get item package folder."""
-        item_folder = Path(
-            self.cwd,
-            "vendor",
-            self.item_id.author,
-            self.item_type_plural,
-            self.item_name,
+        return Path(self.cwd) / ItemRemoveHelper.get_component_directory(
+            PackageId(self.item_type, self.item_id)
         )
-        if not item_folder.exists():
-            # check if it is present in custom packages.
-            item_folder = Path(self.cwd, self.item_type_plural, self.item_name)
-            if not item_folder.exists():
-                raise click.ClickException(
-                    "{} {} not found. Aborting.".format(
-                        self.item_type.title(), self.item_name
-                    )
-                )
-            if (
-                item_folder.exists()
-                and not self.agent_config.author == self.item_id.author
-            ):  # pragma: no cover
-                raise click.ClickException(
-                    "{} {} author is different from {} agent author. "
-                    "Please fix the author field.".format(
-                        self.item_name, self.item_type, self.agent_name
```

Comment:
This check should be maintained. It could be moved under `get_component_directory`?

Reply:
did in `get item config` but it's better to do an overall refactoring, cause component config loading and checks are done in various code places, so the codebase can be unified.